### PR TITLE
[main<] FAQ

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

Updated FAQ.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/help-center/faq.md [hunk 1]
```diff
@@ -132,21 +142,27 @@ required.
...
-* `memgraph/memgraph-platform` - the image contains Memgraph database, Memgraph Lab,
-  mgconsole and MAGE. Once started, mgconsole will be opened in the terminal,
-  while Memgraph Lab is available at `http://localhost:3000`.
-
-The MAGE graph algorithm library includes [NVIDIA
-cuGraph](https://github.com/rapidsai/cugraph) GPU-powered graph algorithms. To
+- `memgraph/memgraph` - the most basic MemgraphDB instance used to run a
+  database instance and a command-line interface.
+- `memgraph/lab` - the image contains a web interface of [Memgraph
+  Lab](https://memgraph.com/docs/data-visualization), a visualization
```
Comment (vpavicic) [inline pages/help-center/faq.md:148]:
```suggestion
  Lab](/data-visualization), a visualization

```

File: pages/help-center/faq.md [hunk 2]
```diff
@@ -132,21 +142,27 @@ required.
...
+  software that helps you explore data stored in Memgraph
+- `memgraph/memgraph-mage` - the image contains a MemgraphDB instance and the
+  command-line interface together with all the newest graph algorithms and
+  utility procedures.
+
+The MAGE graph algorithm library also includes [NVIDIA
+cuGraph](https://github.com/rapidsai/cugraph) GPU-powered graph algorithms. To
 use them, you need a specific kind of `memgraph-mage` image, so check the
-[documentation](/advanced-algorithms/install-mage) or
+[documentation](https://memgraph.com/docs/advanced-algorithms/install-mage) or
```
Comment (vpavicic) [inline pages/help-center/faq.md:157]:
```suggestion
[documentation](/advanced-algorithms/install-mage) or

```

File: pages/help-center/faq.md [hunk 3]
```diff
@@ -132,21 +142,27 @@ required.
...
 [DockerHub](https://hub.docker.com/r/memgraph/memgraph-mage/tags?page=1&name=cugraph)
 for tags.
 
+The easiest way to install Memgraph is to run [Memgraph
+Platform](https://memgraph.com/docs/getting-started) which will run
```
Comment (vpavicic) [inline pages/help-center/faq.md:162]:
```suggestion
Platform](/getting-started) which will run

```

**Review by kgolubic — APPROVED** [review]
Everything looks ok.

**Pull request merged**